_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+* [PEARC](https://pearc.acm.org/pearc22/) Panel - Topic idea: RSE Community
+* [eScience'22](https://www.escience-conference.org/2022/) Workshop - Topic idea: RSEs in Domain Science
+
+Do you have an outreach idea (community, university, event, conference, etc.)? Send an email to [outreach_wg@us-rse.org](mailto:outreach_wg@us-rse.org) or post in the #outreach channel and tag @or-wg!
+
+Starting in January 2022, US-RSE members are meeting biweekly as part of the Outreach Working Group. This WG aims to increase awareness of the US-RSE and RSEng as a profession. The group meets on alternate Fridays at 12 ET/11 CT/10 MT/9 PT. For more information, visit the [Outreach WG webpage](https://us-rse.org/wg/outreach/).
+
+<a name="financials"></a>
+# **US-RSE Financials**
+
+We’re excited to announce that  our t-shirt fundraiser raised $1643.66! The funds will appear in the US-RSE account soon. These funds will go toward the US-RSE [financial goals](https://docs.google.com/presentation/d/19-yTp2Q4SWP5ZiYb3VJa82hflc0p5FkeuhKWlIbwsD4) outlined in the AGM meeting. There are now 112 US-RSE logo shirts and hoodies out there in the wild. We look forward to seeing pictures of them in action around the US-RSE universe and hearing about their new lives and adventures wherever they go.
+
+### Fundraising group starting
+
+Following the awesome success of the t-shirt and  fund-drive we are launching a fund-raising channel on Slack and welcome anyone interested to join. If you are excited about helping raise money toward our current annual $5000/year (or more!) goal please feel free to join the fun. We are open to any and all ideas and expect to organize several fundraising activities during 2022. All the funds will go to the US-RSE organization’s financial goals and mission.
+
+### 2021 financial gift shoutouts
+
+Huge **thanks** to everyone who contributed financially in 2021. In total in 2021 there were 26 gifts to US-RSE. Many thanks to everyone who gave, both named and anonymous financial gifts. Shoutouts to Struan Clark, Ian Cosden, Julia Damerow, Chris Hill, Paul Hsi, Daniel Katz, Joseph H. Kennedy, Constantine Khrulev, Brendan Krueger, Sarom Leang, Christina Maimone, Marshall McDonnell, Lauren Milechin, Rafael Mudafort, Miranda Mundt, David Nicholson, Lance Parsons, Anya Peterson, Luke Rasmussen, Andrea Rubinstein, Vanessa Sochat, Kelly Sovacool, Blaise Thompson, Gaurav Vaidya and Jeff Wagner for your financial contributions.
+
+### See how we are doing financially
+
+We organize US-RSE financials transparently via the [Open Collective](https://opencollective.com/usrse) project. You can see all budgets, expenses, and contributions on [this site](https://opencollective.com/usrse). Anyone can go to the site and see the current accounts, income and expenses and ….. (if the mood takes you and you are able) **[make a gift to US-RSE](https://opencollective.com/usrse#category-CONTRIBUTE)**!
```

Comment:
don't know if we want to somehow mention the (I think!) tax-deductibility here

Reply:
Done!